DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json

```diff
@@ -100,6 +100,7 @@
         },
         "demos": [
             "fTSWithStaticIndexSingleField",
+            "fTSQuerySearchBoosting",
```

Comment:
make the order similar to the C# order
place `fTSQuerySearchBoosting` before the static index demos

Reply:
Done